modules/admin-ui/src/main/webapp/scripts/modules/events/controllers/serieController.js

```diff
@@ -207,24 +207,36 @@ angular.module('adminNg.controllers')
             });
         });
 
-        $scope.theme = {};
+        $scope.selectedTheme = {};
+
+        $scope.updateSelectedThemeDescripton = function () {
```

Comment:
I am no Angular expert, but I see this function called only within this very file. Is there a reason why this lives in the `$scope`? I personally am a friend of free functions for cases like this one, but there might be some Angular best practices/idiosyncrasies I don't know about. Or maybe it is just consistency, again ...? :thinking: